libs/core/algorithms/include/hpx/parallel/container_algorithms/uninitialized_copy.hpp

```diff
@@ -136,9 +136,8 @@ namespace hpx { namespace ranges {
     /// the calling thread.
     ///
     /// \returns  The \a uninitialized_copy algorithm returns an
-    ///           \a in_out_result<typename hpx::traits::range_traits<Rng1>
-    ///           ::iterator_type, typename hpx::traits::range_traits<Rng2>
-    ///           ::iterator_type>.
+    ///           \a in_out_result<typename hpx::traits::range_traits<Rng1>, 
+    ///           typename hpx::traits::range_traits<Rng2>>.
```

Comment:
Same here, the `>::iterator_type` needs to stay. We might want to find a different way of shortening the type. In this case we could write:
```
in_out_result<range_iterator_t<Rng1>, range_iterator_t<Rng2>>
```
instead.